spec/Section 7 -- Response.md

```diff
@@ -135,14 +145,71 @@ locations, where each location is a map with the keys `line` and `column`, both
 positive numbers starting from `1` which describe the beginning of an
 associated syntax element.
 
+If an error can be associated to a particular field in the GraphQL result, it
+should contain an entry with the key `path` that details the path of the
+response field which experienced the error. This allows clients to identify
+whether a `null` result is intentional or caused by a runtime error.
+
+This field should be a list of path segments starting at the root of the
+response and ending with the field associated with the error. Path segments
+that represent object fields should be strings, and path segments that
+represent array indices should be 0-indexed integers. If the error happens
+in an aliased field, the path to the error should use the aliased name, since
+it represents a path in the response, not in the query.
+
+For example, if fetching one of the friends' names fails in the following
+query:
+
+```graphql
+{
+  hero(episode: $episode) {
+    name
+    heroFriends: friends {
+      id
+      name
+    }
+  }
+}
+```
+
+The response might look like:
+
+```js
+{
+  "data": {
+    "hero": {
+      "name": "R2-D2",
+      "heroFriends": [
+        {
+          "id": "1000",
+          "name": "Luke Skywalker"
+        },
+        {
+          "id": "1002",
+          "name": null
+        },
+        {
+          "id": "1003",
+          "name": "Leia Organa"
+        }
+      ]
+    }
+  },
+  "errors": [
+    {
+      "message": "Name for character with ID 1002 could not be fetched.",
+      "locations": [ { "line": 6, "column": 7 }],
+      "path": [ "hero", "heroFriends", 3, "name" ]
```

Comment:
Where is the `3` coming from? My interpretation was that it would indicate what position in the `heroFriends` array had the issue using a zero based index. So for this example I thought it would be `[ "hero", "heroFriends", 1, "name"]`.